util/tlm/SConstruct

```diff
@@ -44,6 +44,7 @@ env = Environment()
 
 #Make the gem5 root available in SConscripts
 env['GEM5_ROOT'] = gem5_root
+env['GEM5BUILD'] = os.path.join(gem5_root, 'build')
```

Comment:
As a note: this won't work with kconfig based builds where the build directory is not necessarily in "build" and can have different names (see https://www.gem5.org/documentation/general_docs/kconfig_build_system/ ). Anyway I am ok on merging this for now as long as we document this issue

Reply:
Gotcha, do I need to do anything specific in terms of documentation aside from having a code comment that specifies that it is not compatible with kconfig builds?